app/services/search_service.rb

```diff
@@ -35,7 +37,11 @@ def perform_accounts_search!
   end
 
   def perform_statuses_search!
-    definition = parsed_query.apply(StatusesIndex.filter(term: { searchable_by: @account.id }))
+    statuses_index = StatusesIndex.filter(term: { searchable_by: @account.id })
+    if SEARCH_ALL_VISIBLE_TOOTS
```

Comment:
Little typo here
```suggestion
    if SEARCH_ALL_VISIBLE_STATUSES
```

Reply:
Yikes, thank you for catching this. It should actually be `SEARCH_ALL_PUBLIC_STATUSES` since we're reducing the scope now.